src/Master.cxx

```diff
@@ -47,6 +77,40 @@ void Master::run() {
   LOG(6, "called stop on all stream_masters");
 }
 
+void Master::statistics() {
+  using namespace rapidjson;
+  rapidjson::Document js_status;
+  js_status.SetObject();
+  auto &a = js_status.GetAllocator();
+  Value js_files;
+  js_files.SetObject();
+  for (auto &stream_master : stream_masters) {
+    auto fwt_id_str =
+        fmt::format("{:016x}", stream_master->file_writer_task().id());
+    auto fwt = stream_master->file_writer_task().stats(a);
+    // TODO
+    // Add status when DM-450 lands.
+    fwt.AddMember(
+        "status",
+        StringRef("[to-be-filled-when-StreamMaster-makes-status-available]"),
+        a);
+    js_files.AddMember(Value(fwt_id_str.c_str(), a), fwt, a);
+  }
+  js_status.AddMember("files", js_files, a);
+  rapidjson::StringBuffer buffer;
+  rapidjson::PrettyWriter<rapidjson::StringBuffer> writer(buffer);
+  writer.SetIndent(' ', 2);
+  js_status.Accept(writer);
+  if (status_producer) {
+    status_producer->produce((KafkaW::uchar *)buffer.GetString(),
+                             buffer.GetSize());
```

Comment:
This casts `size_t` to `int`, make it explicit with a `static_cast` if it is intentional.